<launch>
  <param name="use_sim_time" value="true"/>
  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">

    <rosparam file="$(find navi)/params/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find navi)/params/costmap_common_params.yaml" command="load" ns="local_costmap" />
    
    <rosparam file="$(find navi)/params/map_nav_params/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find navi)/params/map_nav_params/global_costmap_params.yaml" command="load" />
    
    <!-- <rosparam file="$(find navi)/params/base_local_planner_params.yaml" command="load" /> -->
    <!-- <rosparam file="$(find navi)/params/dwa_local_planner_params.yaml" command="load" /> -->
    <rosparam file="$(find navi)/params/teb_local_planner_params.yaml" command="load" />

    <rosparam file="$(find navi)/params/move_base_params.yaml" command="load" />
    
    <param name="base_global_planner" type="string" value="global_planner/GlobalPlanner" />
    
    <!-- <param name="base_local_planner" value="base_local_planner/TrajectoryPlannerROS"/> -->
    <!-- <param name="base_local_planner" value="dwa_local_planner/DWAPlannerROS"/> -->
    <param name="base_local_planner" value="teb_local_planner/TebLocalPlannerROS"/>

    
    <remap from="odom" to="odometry/filtered" />
    
  </node>

  <!-- 启动你的状态机主程序
  <node name="state_machine" pkg="navi" type="main_state_machine.py" output="screen" respawn="false">
    <param name="use_sim_time" value="true" />
  </node> -->
  

</launch>
